src/Kmd.Logic.Gateway.Automation/Client/Generated/GatewayClient.cs

```diff
@@ -7842,7 +7842,7 @@ public async Task<HttpOperationResponse<string>> UpdateRevisionWithHttpMessagesA
                 throw ex;
             }
             // Create Result
-            var _result = new HttpOperationResponse<string>();
+            var _result = new HttpOperationResponse<object>();
```

Comment:
Why auto generated code got changed, is it because of auto rest version change or swagger change ?

Reply:
Because I decided to change only one return parameter, instead of creating a whole new endpoint, just to have a different return value.